### PR TITLE
#943 - added check for correct copyright years (2009 - current)

### LINKED ISSUE #943: check for CI and rultor for years in copyright
Let's add automatic check for Appveyor, Travis, Rultor  for  2009-current year in copyright
see  [Qulice project](https://github.com/teamed/qulice)  for instance

### PR DESCRIPTION
**Pull request by jhyle**
Fixes #943: check for CI and rultor for years in copyright by greping trough all files that have a copyright if the years are set correct and add this to the build scripts

### CODE REVIEW

**Comment by karato** [comment]
@jhyle Many thanks, I will find someone to review it before we merge

**Comment by karato** [comment]
@krzyk review it

**Comment by krzyk** [comment]
@jhyle please add description of what you did in the PR description, also include there reference to the issue you are fixing

File: years.sh [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+if (grep -L -r 2009-`date +%Y` --exclude-dir ".git" --exclude-dir ".settings" --exclude ".*" --exclude-dir "est" --exclude "*.yml" --exclude "*.md" --exclude "*findbug*.xml" --exclude-dir "target" --exclude-dir "node_modules" --exclude-dir "node" . | egrep -v "(src/site/resources/CNAME|netbout-web/src/test|netbout-web/src/main/scss/jqueryTextcomplete.scss|netbout-web/src/main/bower/postinstall.sh|netbout-web/src/main/resources/com/netbout/rest/error.html.vm|netbout-web/src/main/resources/META-INF/MANIFEST.MF|netbout-web/bower.json|src/main/aspect/README.txt|system.properties|Procfile|deploy.sh|years.sh)"); then
+    echo "Files above have wrong years in copyrights"
+    exit 1
+else
```
Comment (krzyk) [inline years.sh]: @jhyle this else is not needed (see http://www.yegor256.com/2015/01/21/if-then-throw-else.html)

**Comment by krzyk** [comment]
@jhyle one additional comment above

**Comment by jhyle** [comment]
@krzyk fixed

**Comment by krzyk** [comment]
@jhyle thanks

**Comment by krzyk** [comment]
@rultor merge pls

**Comment by dmzaytsev** [comment]
@rultor merge

**Comment by karato** [comment]
@elenavolokhova please, review this task for compliance with our [quality rules](http://at.teamed.io/qa.html)

**Comment by karato** [comment]
@rultor deploy pls

**Comment by elenavolokhova** [comment]
@krzyk 
According to our [quality rules](http://www.teamed.io/qa.html):
 > The code reviewer found at least three problems in the code.
 
 There was only one issue found during review.
Please confirm that you will try to find at least three problems next time.

**Comment by krzyk** [comment]
@elenavolokhova I confirm

**Comment by elenavolokhova** [comment]
@karato Quality is acceptable here.

**Comment by karato** [comment]
> @karato Quality is acceptable here.

@elenavolokhova thanks, got it. everybody, please try to make it good next time

**Comment by karato** [comment]
@krzyk 10 mins was added to the account of @elenavolokhova (for QA review), in transaction 75694538; I just added **21 mins** to your account, many thanks for your contribution (75694557).. 4 hours and 46 mins spent here; you're getting extra minutes here (c=6); added +21 to your rating, now it is equal to [+4387](http://www.netbout.com/b/35035?open=rating)

**Pull request merged**